rlclientlib/live_model_impl.cc

```diff
@@ -60,7 +60,7 @@ namespace reinforcement_learning {
     RETURN_IF_FAIL(check_null_or_empty(event_id, context, status));
     if (!_model_data_received) {
       RETURN_IF_FAIL(explore_only(event_id, context, response, status));
-      response.set_model_id("N/A");
+      response.set_model_id("N / A");
```

Comment:
```suggestion
       response.set_model_id("N/A");
```

Reply:
I am ok with this, but all old logs (before flatbuffers change) are having "N / A" - not sure if there is anything (I. e. offline scripts) dependent on this value

Reply:
Oh no, I am wrong - will double check